types/address.go

```diff
@@ -382,6 +383,22 @@ func (va ValAddress) Bytes() []byte {
 	return va
 }
 
+// GetAddress implements the Stringer interface.
+func (va ValAddress) GetAddress() string {
+	if va.Empty() {
+		return ""
+	}
+
+	bech32PrefixValAddr := GetConfig().GetBech32AccountAddrPrefix()
+
+	bech32Addr, err := bech32.ConvertAndEncode(bech32PrefixValAddr, va.Bytes())
+	if err != nil {
+		panic(err)
+	}
+
+	return bech32Addr
+}
+
```

Comment:
This can be remove. Usage can be replaced with `sdk.AccAddress(valAddressInBytes).String()`